pipeline/pipeline/pylintrc

```diff
@@ -5,8 +5,9 @@
 # many local variables, W0603 using the global statement, R0902 too many
 # instance attributes, R1705 unnecessary "else" after "return", W0511 fixme,
 # R0903 Too few public methods, R0401 Cyclic import, R0801 Similar lines
+# W0640 cell-var-from-loop
 
-disable=C0111,W1203,W1202,C0111,R0913,W0622,W0212,W0621,R0914,W0603,R0902,R1705,W0511,R0903,R0401,R0801
+disable=C0111,W1203,W1202,C0111,R0913,W0622,W0212,W0621,R0914,W0603,R0902,R1705,W0511,R0903,R0401,R0801,W0640
```

Comment:
if we make copy_task_inputs an instance method somehow, then we don't have to ignore W0640. I think W0640 is generally a good warning though I understand why you added it. I prefer it's a line-specific ignore though.